x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/handle_api_errors.ts

```diff
@@ -33,7 +33,7 @@ interface Options {
   isQueued?: boolean;
 }
 
-export const defaultErrorMessage = i18n.translate(
+export const defaultErrorMessage: string = i18n.translate(
```

Comment:
nitpick: doesn't need to be declared as a string type, should be able to infer.

Reply:
I think @TattdCodeMonkey added this and he had some explanation, but I might be wrong.

Reply:
@joemcelroy this was to fix a TS error when using this value. TS was having a hard time knowing all possible results from `getErrorsFromHttpResponse()` without this being explicitly typed as a `string`